cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/interactor-tooltips.tsx

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+import Image from 'antd/lib/image';
+import Paragraph from 'antd/lib/typography/Paragraph';
+import Text from 'antd/lib/typography/Text';
+
+interface Props {
+    tool?: string;
+    withNegativePoints?: boolean;
+}
+
+function InteractorTooltips(props: Props): JSX.Element {
+    const { tool, withNegativePoints } = props;
+
+    const DEXTR_GIF = 'https://openvinotoolkit.github.io/cvat/images/dextr_example.gif';
```

Comment:
Probably better to add that into function.yaml for corresponding serverless functions. UI should not know about specific serverless functions. What do you think?

Reply:
@nmanovic 

Agree. Just in this case need to redeploy the interactors again

Reply:
On the other hand, we should not store the full path on interactor (only image name maybe). To support fetching gif from specific documentation releases/develop.

Reply:
Need to redeploy, but it is a small pay for a good design choice. Users will be able to add "documentation" for own DL models and UI will show it automatically.

I believe the interactor should be responsible for the image location. We can get it even from another site (e.g. https://raw.githubusercontent.com/shiyinzhang/Inside-Outside-Guidance/master/ims/IOG.gif). At the same time for an image from official documentation we can have a shortcut. Probably better to insert a link with a specific comment to avoid problems with versions: https://raw.githubusercontent.com/openvinotoolkit/cvat/0fbb19ae3846a017853d52e187f0ce149adced7d/site/content/en/images/dextr_example.gif

Reply:
Done